swablu/tpl/jam.html

```diff
@@ -1,4 +1,19 @@
 {% include 'head.html' %}
+{% if cannot_vote and cannot_vote == "1" %}
+<div class="red">
+    <span>Error: Voting is not currently active for this jam. Your vote was not saved.</span>
+</div>
+{% end %}
+{% if cannot_vote and cannot_vote == "2" %}
+<div class="red">
+    <span>Error: Jury members cannot cast community votes. Your vote was not saved.</span>
+</div>
+{% end %}
+{% if cannot_vote and cannot_vote == "3" %}
```

Comment:
3 is the current fallback value (see line 403). I guess this could be improved a bit with something like
```
if (cannot_vote) {
    if (cannot_vote == 1) {
        // Error message 1
    } else if (cannot_vote == 2) {
        // Error message 2
    } else {
        // Default error message
    }
}
```
But I don't know what the syntax for that is, nor if `if` blocks can be nested.